src/main/java/org/folio/dataexp/repository/JobProfileEntityRepository.java

```diff
@@ -9,6 +9,6 @@
 
 public interface JobProfileEntityRepository extends JpaRepository<JobProfileEntity, UUID> {
 
-  @Query(value = "SELECT id FROM job_profiles WHERE jsonb ->> 'default' = 'true' AND jsonb ->> 'name' like '%'||:name||'%'", nativeQuery = true)
+  @Query(value = "SELECT id FROM job_profiles WHERE jsonb ->> 'default' = 'true' AND jsonb ->> 'name' like 'Default%'||:name||'%'", nativeQuery = true)
```

Comment:
Hi @obozhko-folio , check please if this logic will be correct if some one will use name parameter starting from 'Default'

Reply:
if someone will rename in DB?

Reply:
In the code only RecordTypeEnum or IdTypeEnum is used as a name parameter, 'Default' is not used.

Reply:
I'm happy that smth in code is used, but you check value in DB, not in code

Reply:
If someone renames it in DB, we will never find it. All job profiles are hardcoded.

Reply:
in which places do we hardcode name as well? AFAIK it's the first one where you try to introduce such query

if you don't want to fix base on regular comment, I can request changes to not use hardcode in SQL, try to find other ways to solve it

Reply:
Here is a folder where all default job profiles are hardcoded: [job profiles](https://github.com/folio-org/mod-data-export/tree/master/src/main/resources/data/job-profiles)